src/documents.rs

```diff
@@ -117,6 +120,30 @@ impl<'a, Http: HttpClient> DocumentQuery<'a, Http> {
         self
     }
 
+    /// Return document vector data with search result.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # use meilisearch_sdk::{client::*, indexes::*, documents::*};
+    /// #
+    /// # let MEILISEARCH_URL = option_env!("MEILISEARCH_URL").unwrap_or("http://localhost:7700");
+    /// # let MEILISEARCH_API_KEY = option_env!("MEILISEARCH_API_KEY").unwrap_or("masterKey");
+    /// #
+    /// # let client = Client::new(MEILISEARCH_URL, Some(MEILISEARCH_API_KEY)).unwrap();
+    /// let index = client.index("document_query_with_fields");
```

Comment:
```suggestion
    /// let index = client.index("document_query_with_retrieve_vectors");
```

If you re-use the same index everywhere your test will be flaky